examples/angular/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "private": true,
   "dependencies": {
+    "@aws-amplify/ui-angular": "0.0.1",
```

Comment:
Adding explicit dependency to `@aws-amplify/ui-angular` here. Will be bumped to "2.0.1" on changesets pr later this week.